### ISSUE TITLE
How to get an unqualified url?

### ISSUE THREAD

**Issue by ccorcos** [unshiftio/url-parse#78]
If I have a url like `https://www.example.com/abc?q=10#hello` then I can use `parsed.href.slice(parsed.origin.length)` to get just `/abc?q=10#hello`. However, this doesnt work if I create parsed using the unqualified url because origin will be "null".

I see another discussion going on about "null" but regardless, I think it would be nice if there was a convenient way of getting an unqualified url using this library.

**Comment by lpinca** [comment]
I'm not sure I understand. If you use a relative URL, isn't the whole URL what you are looking for?

**Comment by ccorcos** [comment]
Yes. But it's prone to error if I have to put that if statement everything. People who are unfamiliar with this library will see `=== 'null'` and thing wtf?

**Comment by lpinca** [comment]
You can use a `baseUrl` in that case.
